grafana/grafana
# Traces: Add more template variables in Tempo & Zipkin

**What this PR does / why we need it**:

This PR adds more template variables in Tempo and Zipkin and allows them to be interpolated to explore.

**How to test**:

1. Create ds (Tempo, Zipkin)
2. Add template vars
3. Open up network tab and ensure query is using interpolated values
4. Click on saved dashboard panels header and click explore
5. Ensure variables are interpolated in explore
